### ISSUE TITLE
NetworkPolicy ingress port is not configurable

### ISSUE THREAD

**Issue by alita1991** [bitnami-labs/sealed-secrets#1220]
**Which component**:
controller:v0.20.2 (image version)
sealed-secrets:2.8.1 (chart version)

**Describe the bug**
While looking at how to enable NetworkPolicy on the sealed-secrets deployment, I noticed that the NetworkPolicy Ingress port number is hardcoded in the chart. This can generate an issue if I need to change the port number, like in my case.

```
{{- if .Values.networkPolicy.enabled }}
apiVersion: {{ include "sealed-secrets.networkPolicy.apiVersion" . }}
kind: NetworkPolicy
metadata:
  name: {{ include "sealed-secrets.fullname" . }}
  namespace: {{ include "sealed-secrets.namespace" . }}
  labels: {{- include "sealed-secrets.labels" . | nindent 4 }}
spec:
  podSelector:
    matchLabels: {{- include "sealed-secrets.matchLabels" . | nindent 6 }}
  ingress:
    - ports:
      - port: 8080
{{- end }}
```

**To Reproduce**
1. Enable NetworkPolicy
2. Change the 8080 port value to 8081
3. Check the NetworkPolicy Ingress port value

**Expected behavior**
NetworkPolicy Ingress port value should be configurable.

**Version of Kubernetes**:
v1.26

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"26", GitVersion:"v1.26.3", GitCommit:"9e644106593f3f4aa98f8a84b23db5fa378900bd", GitTreeState:"clean", BuildDate:"2023-03-15T13:40:17Z", GoVersion:"go1.19.7", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.7
Server Version: version.Info{Major:"1", Minor:"26", GitVersion:"v1.26.0+k3s1", GitCommit:"fae8817655a8ad1250d40e5b4f9a938cbb9c960a", GitTreeState:"clean", BuildDate:"2022-12-21T00:22:30Z", GoVersion:"go1.19.4", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
I'm currently trying to align the charts I use to the recommended community best-practices.

**Comment by alita1991** [comment]
In addition to this, I notice that egress is not defined. Can we add a default egress for it as well?

**Comment by Cajga** [comment]
@agarcia-oss Would be nice to include the egress as well. Sealed Secrets would only require to talk to the API only (to allow it to subscribe to the CRDs), or?

**Comment by alvneiayu** [comment]
hi everyone, issue resolved by #1243.
